packages/truffle/cli.webpack.config.js

```diff
@@ -56,8 +56,14 @@ module.exports = {
   devtool: "source-map",
   module: {
     rules: [
-      { test: /\.js$/, use: "shebang-loader" },
-      { test: /rx\.lite\.aggregates\.js/, use: "imports-loader?define=>false" }
+      {
+        test: /\.js$/,
+        include: [
+          path.resolve(__dirname, "../core"),
+          path.resolve(__dirname, "../environment")
```

Comment:
Maybe it's worth a comment above each of these to say why we're using the shebang-loader in that package.